language-support/qute/qute-json.tmLanguage.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Qute JSON",
+  "injections": {
+    "L:support.type.property-name.json, L:string.quoted.double.json": {
+      "patterns": [
+        {
+          "include": "grammar.qute#templates"
+        },
+        {
+          "include": "grammar.qute#comment"
+        }
+      ]
+    },
+    "L:meta.structure.dictionary.json": {
+      "patterns": [
+        {
+          "include": "grammar.qute#comment"
+        }
+      ]
+    }
+  },
+  "patterns": [
+    {
+      "include": "source.json"
+    }
+  ],
+  "scopeName": "source.json.qute",
+  "uuid": "C011BB07-875D-4DEB-9582-0E1FEC0D1E6E"
```

Comment:
see https://stackoverflow.com/a/59791552/753170

Reply:
yes, that was me who asked that :)